packages/core/src/package-name.ts

```diff
@@ -1,5 +1,5 @@
 export default function absolutePackageName(specifier: string): string | undefined {
-  if (specifier[0] === '.' || specifier[0] === '/') {
+  if (specifier[0] === '.' || specifier[0] === '/' || specifier.startsWith('C:')) {
```

Comment:
This change fixes a new error:

```

Build Error (PackagerRunner) in app.js

Module build failed (from C:/Users/thoover/Desktop/embroider-windows/node_modules/babel-loader/lib/index.js):
Error: $TMPDIR\embroider\f97ddc\app.js: $TMPDIR\embroider\externals\C:\Users\thoover\Desktop\embroider-windows\node_modules\@babel\runtime\helpers\esm contains invalid WIN32 path characters.
```

Reply:
I don’t think we can hard code `C:\` here. It is perfectly valid to use other drive names...

Reply:
Let's use path.isAbsolute instead of the C: or `/` checks:

https://nodejs.org/api/path.html#path_path_isabsolute_path

Reply:
Definitely agree with @rwjblue 👍

Reply:
@rwjblue Good catch! Updated